Makefile

```diff
@@ -52,6 +52,13 @@ again:
 	MODE=sushi cargo test --features prove_verify --release test_evm_prove_verify > $(CURRENTDATE).sushi.evm.txt 2>&1; \
 	MODE=sushi cargo test --features prove_verify --release test_state_prove_verify > $(CURRENTDATE).sushi.state.txt 2>&1
 
+scroll:
+	cargo build --release
+	cp target/release/libservice.so ../scroll/verifier/lib/libverifier.so
+
+roller:
+	cargo build --release
+	cp target/release/libservice.so ../go-roller/roller/prover/lib/libprover.so
```

Comment:
```suggestion
bin:
	cargo build --release

prepare_for_scroll: bin
	cp target/release/libservice.so ../scroll/verifier/lib/libverifier.so

prepare_for_roller: bin
	cp target/release/libservice.so ../go-roller/roller/prover/lib/libprover.so
```

Reply:
I cannot really find `target/release/libservice.so` after, `cargo build --release`, it seems this Makefile is out-of-date?

Reply:
Yeah, it is out-of-date now. I moved `lib verifier`and `lib prover` into `scroll` and `go-roller`. I will remove these makefile cmd.